servicebus/data-plane/azure-servicebus/src/test/java/com/microsoft/azure/servicebus/TestBase.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.microsoft.azure.servicebus;
+
+import com.azure.common.test.TestMode;
+import org.junit.Assume;
+import org.junit.BeforeClass;
+
+public class TestBase {
+
+    @BeforeClass
+    public static void skipIfNotConfigured() {
+        Assume.assumeTrue("The test only runs in Live mode.", TestBase.getTestMode() == TestMode.RECORD);
+    }
+
+    public static TestMode getTestMode() {
+        String testMode = System.getenv("AZURE_TEST_MODE");
+        if (testMode == null) {
```

Comment:
What if someone declares AZURE_TEST_MODE == record or AZURE_TEST_MODE == "does not exist"? You should try to actually parse the environment variable. Also, we should have AZURE_TEST_MODE constant defined in a common class, so we aren't declaring/possibly changing the variable name.

Reply:
updated